src/current/v25.3/automatic-go-execution-tracer.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Automatic Go Execution Tracer
+summary: The automatic Go execution tracer can be used to gain fine-grained insight into goroutine scheduling, blocking events, garbage collection, and runtime activity.
+keywords: go execution trace, go execution tracing, execution tracer, automatic execution tracer
+toc: true
+---
+
+The automatic Go execution tracer allows operators and support engineers to investigate internal system behavior over time. Use this feature to gain fine-grained insight into goroutine scheduling, blocking events, garbage collection, and runtime activity.
+
+{{site.data.alerts.callout_info}}
+Execution tracing introduces a performance overhead. This feature is primarily intended for use under the guidance of [Cockroach Labs Support](https://support.cockroachlabs.com/).
+{{site.data.alerts.end}}
+
+When enabled, the automatic Go execution tracer captures runtime execution traces on a recurring schedule and stores them in the node's log directory. These traces are collected in the background and can be analyzed using Go’s standard tooling (e.g., [`go tool trace`](https://pkg.go.dev/runtime/trace)).
+
+To prevent disk overuse, older traces are automatically garbage collected based on a configurable storage limit.
+
+## Configuration
+
+You can configure automatic Go execution trace capture with the following [cluster settings]({% link {{ page.version.version }}/cluster-settings.md %}):
+
+Cluster Setting | Description | Default Value
+----------------|-------------|---------------
+`obs.execution_tracer.interval` | Enables periodic execution tracing on each node and defines how often traces are collected. Set to a positive duration (e.g., `10m`, `1h`) to enable. Set to `0` to disable (default). | `0`
+`obs.execution_tracer.duration` | Sets how long each execution trace runs. Use short durations (e.g., 5-15 seconds) to produce traces that are easier to analyze. | `10s`
+`obs.execution_tracer.total_dump_size_limit` | Sets a soft limit on the total disk space used by execution trace files. When the limit is exceeded, the oldest files are deleted. | `4GiB`
```

Comment:
"are deleted" or "may be deleted"?

(this sort of thing comes up all the time with MVCC GC TTL settings)